build.sbt

```diff
@@ -122,23 +125,35 @@ lazy val core = (project in file("."))
       "co.fs2" %% "fs2-core" % fs2Version,
       "co.fs2" %% "fs2-io" % fs2Version,
       "com.google.protobuf" % "protobuf-java" % "3.25.1",
-      "org.tpolecat" %% "natchez-core" % natchezVersion
+      "org.tpolecat" %% "natchez-core" % natchezVersion,
+      "org.scala-lang.modules" %% "scala-collection-compat" % "2.12.0"
```

Comment:
will remove it back, not sure it's correct to pull 2.12.0 to 2.13 and 3 builds, also there's another way to suppress warnings depending on versions